README.md

```diff
@@ -31,7 +31,7 @@ Deploy and log in with username *admin* and the password that's set up during de
 
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3)
- * PHP <b>7.3</b> or higher
+ * PHP <b>7.4</b> or higher
  * [Composer](https://getcomposer.org/) <b>1.4</b> or higher
  * NodeJS <b>10.13.0</b> or higher
```

Comment:
@driusan all these rows were part of the conflict. i only changed the PHP version